### PR TITLE
xcb-util

### PR DESCRIPTION
**Pull request by scopatz**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by scopatz** [comment]
@conda-forge/staged-recipes - this is a linux-only recipe that is needed for the kwindowsystem package

**Comment by scopatz** [comment]
Thanks!

File: recipes/xcb-util/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - make
+    - pkg-config
+  host:
+    - libxcb >=1.13
+    - xorg-xproto
+  run:
+    - libxcb >=1.13
```
Comment (isuruf) [inline recipes/xcb-util/meta.yaml]: This should be something like,
```
- {{ pin_compatible("libxcb", max_pin=None) }}
```
for the correct lower bound.

Comment (beckermr) [reply recipes/xcb-util/meta.yaml]: Ahhh sure. This makes for less maintenance and only one version in the recipe. Ideally libxcb would have a run export.

Comment (isuruf) [reply recipes/xcb-util/meta.yaml]: This is not for maintenance. This is for correctness.
For example, xcb-util might be built against 1.14, but the run is still >=1.13, but we want it to be >=1.14

Comment (beckermr) [reply recipes/xcb-util/meta.yaml]: Ah I see your point. This assumes something might happen with the solver picking two different versions, which is totally possible. Easy to clean up in the feedstock.

**Pull request merged**